matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/dtos/offers/PositionsRequest.java

```diff
@@ -42,19 +43,19 @@ public String resourcePath() {
     @Override
     public Map<String, String> parameters() {
         Map<String, String> parameters = new HashMap<>();
-        if (!eventIds.isEmpty()) {
+        if (Objects.nonNull(eventIds) && !eventIds.isEmpty()) {
```

Comment:
If I'm not wrong @zupermanzupereroe fixed this issue in a different way, similar to other requests. He initialized the collections in the builder and used empty lists when parameters are not specified.

Reply:
I would suggest to handle these cases the same way in all implementations of requests classes.

Reply:
yeah, i agree with you that it should be the same in all the classes. I did that PR to show how i believe it should be implemented. 

In my opinion, will be better to check is objects is not null, that create an empty object in a perspective of garbage producing

Reply:
I agree null-checking is better than empty collection by default. We'll change this in every place in a future commit.